apps/frontend/src/libs/components/content/libs/constants.ts

```diff
@@ -0,0 +1,4 @@
+const MAXIMUM_LENGTH = 400;
```

Comment:
```suggestion
const MAXIMUM_CONTENT_LENGTH = 400;
```
I think that you need to name it more detailed, because maximum length is not so obvious